### ISSUE TITLE
VoiceOver "Load More" button in timeline has no default activation

### ISSUE THREAD

**Issue by alexbbrown** [metabolist/metatext#250]
**Describe the bug**
a customer reports on the app store: 

> the "load more" button in the timeline view can't be activated while using Voiceover

**To Reproduce**
i have reproduced the issue; when the "load more " button is selected in voiceover, the default activation that is executed by double click does nothing.

I also observed that if you switch the VoiceOver rotor to action mode, two more actions are available: 
* load from toot above
* Load from toot below

**Expected behavior**
the button should have a default action. 

 If it's unsafe to automatically pick one of the two actions, then I recommend that the VoiceOver alternative be represented as two buttons, one for each relevant action.

**Screenshots**
see thread below

**(please complete the following information):**
 - Device: iPhone 12
 - OS: iOS 16.1.1
 - App Version: 1.7 

**Additional context**
the full review from the app store

> I am very happy with this app as a Voiceover user. It's functionality is excellent. Unlike the official Mastodon app, this app includes rotor actions that make using it much faster and easier. I have found only two bugs in the app with Voiceover. First, the "load more" button in the timeline view can't be activated while using Voiceover. Second, when in your list of following or followers, you can't activate entries to go to the person's profile page after scrolling past the first few entries in the list. Neither bug is a huge deal though. The only thing that would be nice as an addition would be the ability to view the public timelines of instances you aren't signed into.
Thanks so much to the dev for making such a great and accessible app available for free.

I have documented the follower issue in #249

**Comment by alexbbrown** [comment]
Video showing the toot feed including the load more button, and then activating the action rotor and showing two more actions.  

https://user-images.githubusercontent.com/2532344/204107240-04d9a2c7-1b94-4b82-9f5e-d0ac8e6ec02c.mov

**Comment by alexbbrown** [comment]
Here's the state when I double tapped but nothing happened

![IMG_0206 screenshot showing the toot feed, with a button labelled load more with two arrows pointing up.  At the bottom a caption reads "Load More"](https://user-images.githubusercontent.com/2532344/204107294-ef3816de-ab64-46f1-a15b-07600f3d89e9.png)
